handsontable/test/e2e/core/selectRows.spec.js

```diff
@@ -48,17 +48,23 @@ describe('Core.selectRows', () => {
       `).toBeMatchToSelectionPattern();
   });
 
-  it('should mark non-contiquous selection when CTRL key is pressed', () => {
+  // TODO: After changes introduced in Handsontable 12.0.0 we handle shortcuts only by listening Handsontable.
+  // Please keep in mind not passing test is consequence of bug https://github.com/handsontable/handsontable/issues/7290.
+  xit('should mark non-contiquous selection when CTRL key is pressed', () => {
```

Comment:
```suggestion
  it('should mark non-contiquous selection when CTRL key is pressed', () => {
```
See here https://github.com/handsontable/handsontable/pull/8955/files#r810947881

Reply:
As in https://github.com/handsontable/handsontable/pull/8955#discussion_r811899513.